src/core_landice/mode_forward/mpas_li_thermal.F

```diff
@@ -1079,6 +1082,14 @@ subroutine li_thermal_solver(domain, err)
                           basalEnthalpy,             &
                           basalTemperature(iCell))
 
+                    
```

Comment:
I think there may be extra whitepspace that was added to this line and should be removed.

Reply:
Removed this empty line